.github/workflows/all_plugins.yaml

```diff
@@ -33,6 +33,23 @@ jobs:
           melos exec -c 1 --no-private --ignore="*example*" -- \
             pub publish --dry-run
 
+  pub_get_check:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+      - name: "Install Flutter"
+        run: ./.github/workflows/scripts/install-flutter.sh stable
```

Comment:
If you don't wrap these in https://github.com/nick-invision/retry/ you will likely be flaky